packages/addon-dev/src/rollup-hbs-plugin.ts

```diff
@@ -19,5 +23,39 @@ export default function rollupHbsPlugin(): Plugin {
         id: id + '.js',
       };
     },
+
+    // template-only components may be imported
+    async resolveId(source, importer, options) {
+      if (!options.isEntry) {
```

Comment:
I think a template-only component could be an entrypoint, which might make options.isEntry true. Can you test that case -- an addon that wants to use a template-only component in publicEntrypoints and appReexports?

Reply:
yeah, I'll add something to the demo repo! thanks!